docs/src/index.md

```diff
@@ -43,52 +43,51 @@ obs_func = nothing
 # now garbage collection can at any time clear the connection
 ```
 
-### Async operations
+### Priority
 
-#### Delay an update
+One can also give the callback a priority, to enable always calling a specific callback before/after others, independent of the order of registration.
```

Comment:
Somewhere, one should specify how to interpret the priority: is it always an `Int`? Which occurs first, minimum priority or maximum priority? (Does -1 happen before or after 1?)